tests/test_umath.py

```diff
@@ -88,8 +72,15 @@ def get_id(val):
 @pytest.mark.parametrize("test_cases", test_cases, ids=get_id)
```

Comment:
A few warnings are generated, you may want to add
`@pytest.mark.usefixtures("suppress_divide_invalid_numpy_warnings")`

tests/test_umath.py::test_umaths[('arccos', 'f')] /home/vtavanas/repos/dpnp/tests/test_umath.py:86: RuntimeWarning: invalid value encountered in arccos
  expected = getattr(numpy, umath)(*args)
PASSED
tests/test_umath.py::test_umaths[('arccosh', 'f')] /home/vtavanas/repos/dpnp/tests/test_umath.py:86: RuntimeWarning: invalid value encountered in arccosh
  expected = getattr(numpy, umath)(*args)
PASSED
tests/test_umath.py::test_umaths[('arcsin', 'f')] /home/vtavanas/repos/dpnp/tests/test_umath.py:86: RuntimeWarning: invalid value encountered in arcsin
  expected = getattr(numpy, umath)(*args)
PASSED
tests/test_umath.py::test_umaths[('arctanh', 'f')] /home/vtavanas/repos/dpnp/tests/test_umath.py:86: RuntimeWarning: divide by zero encountered in arctanh
  expected = getattr(numpy, umath)(*args)
/home/vtavanas/repos/dpnp/tests/test_umath.py:86: RuntimeWarning: invalid value encountered in arctanh
  expected = getattr(numpy, umath)(*args)
PASSED

Reply:
Yep, great idea. Thank you!